modules/aggregation-layers/package.json

```diff
@@ -16,13 +17,19 @@
     "type": "git",
     "url": "https://github.com/visgl/deck.gl.git"
   },
-  "main": "dist/es5/index.js",
-  "module": "dist/esm/index.js",
-  "types": "typed/index.d.ts",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.cjs",
+  "module": "dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js",
+      "require": "./dist/index.cjs"
+    }
```

Comment:
Minor — I think this could be simplified to...

```suggestion
    "types": "./dist/index.d.ts",
    "import": "./dist/index.js",
    "require": "./dist/index.cjs"
```

... and it'd be the same thing, any reason to keep the `.` wrapper?

Reply:
`exports` is used by node for package resolution: https://nodejs.org/api/esm.html#resolution-and-loading-algorithm

Reply:
I don't think I follow what you mean — these should be equivalent exports, in both bundlers and node.js module resolution:

```jsonc
"exports": {
  ".": {
    "types": "./dist/index.d.ts",
    "import": "./dist/index.js",
    "require": "./dist/index.cjs"
  }
}
```

```jsonc
"exports": {
  "types": "./dist/index.d.ts",
  "import": "./dist/index.js",
  "require": "./dist/index.cjs"
}
```

Reply:
Not 100% on this, but I was under the impression that there is a difference between "." and "./" in that the first disallows cherry picking sub file imports `import ... from '@deck.gl/core/lib/utils'` and the slash version does allow this. Not quite sure which semantics omitting it represents.

Reply:
Oh! If it has the effect of preventing subdirectory imports, I wasn't aware of that. I've also seen wildcard matching:

https://github.com/mrdoob/three.js/blob/841d2e791d3e8a2463322c5ca31b16956828b91c/package.json#L9-L18

In any case... my suggestion of removing `"."` was only aesthetic. So if there could be functional differences, or we think we might want to add other exports later, then perhaps keeping these `"."` blocks would be best. Please ignore my suggestion above. :)